sdk/core/Microsoft.Azure.Core.NewtonsoftJson/src/NewtonsoftJsonETagConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Azure.Core.Serialization
+{
+    /// <summary>
+    /// A <see cref="JsonConverter"/> implementation for <see cref="ETag"/>.
+    /// </summary>
+    public class NewtonsoftJsonETagConverter: JsonConverter
```

Comment:
Should we add this to `NewtonsoftJsonObjectSerializer` by default?

Reply:
Added, also added a method to get the default settings in case customers want to change a few simple things.